### ISSUE TITLE
Multiple Boards

### ISSUE THREAD

**Issue by 14mane88** [kscholty/SmartShuntINA226#5]
Hey, would it be possible to add more than one INA226 Board to the Wemos/ Software. I have 3 Batteries and i'd like to monitor them by 1 Wemos. Is there any possibility to implement it?

**Comment by mrpeter322** [comment]
Yes, just use muliple ina266 modules. For second look on the other side of the module and connect A0 with VCC and for third connext A1 and VCC. Then you can connect them on same i2c and there will not be a address fight. Feel free to choose random alarm pin (based on your board docs).
![image](https://github.com/kscholty/SmartShuntINA226/assets/173613592/fd482eab-5ea4-4002-aef5-fa90382d29b5)
